### ISSUE TITLE
Warnings about urls.py with Django 1.9

### ISSUE THREAD

**Issue by rudyryk** [TigorC/django-redactorjs#12]
```
... site-packages/redactor/urls.py:20: RemovedInDjango110Warning:
django.conf.urls.patterns() is deprecated and will be removed in Django 1.10.
Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  name='redactor_upload_file'))
```

Seems like `urls.py` needs to use a conditional definition of `urlpatterns` depending on `django.__version__`.

**Comment by derekolsson** [comment]
Created pull request with fix for 1.10

**Comment by TigorC** [comment]
Fixed in #13
